tutorials/plot_object_annotations.py

```diff
@@ -0,0 +1,141 @@
+"""
+The **events** and :class:`Annotations <mne.Annotations>` data structures
+=========================================================================
+
+Events and annotations are quite similar. This tutorial highlights their
+differences and similitudes and tries to shade some light to which one is
+preferred to use in different situations when using MNE.
+Here follows both terms definition from the :ref:`glossary`.
+
+    events
+        Events correspond to specific time points in raw data; e.g., triggers,
+        experimental condition events, etc. MNE represents events with integers
+        that are stored in numpy arrays of shape (n_events, 3). Such arrays are
+        classically obtained from a trigger channel, also referred to as stim
+        channel.
+
+    annotations
+        One annotation is defined by an onset, a duration and a string
+        description. It can contain information about the experiments, but
+        also details on signals marked by a human: bad data segments,
+        sleep scores, sleep events (spindles, K-complex) etc.
+
+They both can be seen as triplets where the first element answers to **when**
+something happens and the last element refers to **what** is it. The
+main differnce is that events the when is samples with respect to the first
+sample and the what is an integer id; while in annotations the when is in
+seconds with respect to an origin and the what is an arbitrary string.
+The second element of the triplets have no direct relation between the two
```

Comment:
There is no correspondence between the second element of events and annotations. For events, the second element corresponds to the ID of the previously active event. The second element of :class:`mne.Annotations` is a float indicating its duration in seconds.